cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -86,87 +85,97 @@ Kitematic and then download, start, and connect to an instance of CDAP.
+   above (such as ``192.168.99.100``) as the host name when either connecting to the CDAP
+   UI or making an HTTP request.
+
+#. Once Docker has started, pull down the *CDAP Docker Image* from the Docker Hub using:
+
+   .. tabbed-parsed-literal::
+     :tabs: "Linux or Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+ 
+     $ docker pull caskdata/cdap-standalone:|release|
 
-- Start the *Docker CDAP Virtual Machine* with:
+#. Start the *Docker CDAP Virtual Machine* with:
 
-  .. tabbed-parsed-literal::
-    :tabs: "Mac OS X",Windows
-    :mapping: linux,windows
-    :dependent: linux-windows
-    :languages: console,shell-session
+   .. tabbed-parsed-literal::
+     :tabs: "Linux or Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+ 
+     $ docker run -t -i -p 9999:9999 -p 10000:10000 caskdata/cdap-standalone:|release|
 
-    .. Mac OS X
-    
-    $ docker run -t -i -p 9999:9999 -p 10000:10000 caskdata/cdap-standalone:|release|
-    
-    .. Windows
-    
-    > docker run -t -i -p 9999:9999 -p 10000:10000 caskdata/cdap-standalone:|release|
+#. CDAP will start automatically once the CDAP Virtual Machine starts. CDAP’s software
+   directory is under ``/opt/cdap/sdk``.
 
-- CDAP will start automatically once the CDAP Virtual Machine starts. CDAP’s Software
-  Directory is under ``/opt/cdap/sdk``.
+#. Once CDAP starts, it will instruct you to connect to the CDAP UI with a web browser
+   at ``http://localhost:9999``. 
+  
+#. If you are **running Docker on either Mac OS X or Microsoft Windows**, replace ``localhost`` 
+   with the Docker VM's IP address (such as ``192.168.99.100``) that you obtained earlier.
+   Start a browser and enter the address to access the CDAP UI.
+
+#. To control the CDAP instance, use this command, substituting one of ``start``, ``restart``,
+   or ``stop`` for ``<command>``:
+
+   .. tabbed-parsed-literal::
+     :tabs: "Linux or Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+     
+     $ docker exec -d $(docker ps | grep cdap-standalone | awk '{print $1}') /opt/cdap/sdk/bin/cdap.sh <command>
+  
+#. When you are finished, stop CDAP and shutdown Docker:
 
-- Once CDAP starts, it will instruct you to connect to the CDAP UI with a web browser
-  at ``http://localhost:9999``. Replace ``localhost`` with the Docker VM's IP address 
-  (such as ``192.168.59.103``) that you obtained earlier. Start a browser and enter the
-  address to access the CDAP UI.
+   .. tabbed-parsed-literal::
+     :tabs: "Linux or Mac OS X",Windows
+     :mapping: linux,windows
+     :dependent: linux-windows
+     :languages: console,shell-session
+     
+     $ docker exec -d $(docker ps | grep cdap-standalone | awk '{print $1}') /opt/cdap/sdk/bin/cdap.sh stop
+     $ docker-machine stop cdap
```

Comment:
But there's no VM on Linux... I'm pretty sure that binary doesn't even exist on my Linux install. You can get a `docker-base` cluster from Coopr for testing.